recipes/date/all/test_package/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def is_sudo_enabled(self):
+        if "CONAN_SYSREQUIRES_SUDO" not in os.environ:
+            if os.name == 'posix' and os.geteuid() == 0:
+                return False
+            if os.name == 'nt':
+                return False
+        return os.getenv("CONAN_SYSREQUIRES_SUDO", True)
+
+    def copy_utc(self):
+        if tools.os_info.is_linux:
```

Comment:
what's very complicated, and you're trying to install something to `/etc` or `/usr`. is there a way to read from the specified file instead?